dallinger/command_line/lib/ec2.py

```diff
@@ -684,7 +688,7 @@ def stop(region_name, instance_id):
     logger.info(f"Stopping of {instance_id} complete!")
 
 
-def teardown(region_name, instance_id, public_dns_name, dns_host):
+def teardown(region_name, instance_id, public_dns_name, dns_host, name=None):
```

Comment:
I would prefer the parameter to be called `instance_name` here as you have other names already in the call.

I am confused about why `name` is optional? Doesn't the user always have to specify it?

Reply:
This part of the code has been modified